src/main/java/stsjorbsmod/cards/Feast.java

```diff
@@ -0,0 +1,50 @@
+package stsjorbsmod.cards;
+
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.RememberSpecificMemoryAction;
+import stsjorbsmod.characters.Wanderer;
+import stsjorbsmod.memories.GluttonyMemory;
+
+import static stsjorbsmod.JorbsMod.makeCardPath;
+import static stsjorbsmod.characters.Wanderer.Enums.REMEMBER_MEMORY;
+
+// 1: Remember Gluttony. Draw 3(4) cards. Exhaust.
+public class Feast extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Feast.class.getSimpleName());
+    public static final String IMG = makeCardPath("Manipulation_Rares/feast.png");
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Wanderer.Enums.COLOR_GRAY;
+
+    private static final int COST = 1;
+    private static final int DRAW = 3;
+    private static final int UPGRADE_PLUS_DRAW = 1;
+
+    public Feast() {
+        super(ID, IMG, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = DRAW;
+        exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        enqueueAction(new RememberSpecificMemoryAction(new GluttonyMemory(p, false)));
+        enqueueAction(new DrawCardAction(p, magicNumber, false));
+
+        tags.add(REMEMBER_MEMORY);
+    }
+
+    @Override
+    public void upgrade() {
+        if (!upgraded) {
+            upgradeName();
+            upgradeMagicNumber(UPGRADE_PLUS_DRAW);
+            //initializeDescription();
```

Comment:
Is this intended to be commented out?

Reply:
I'll remove it for now. It seems like most upgrade() overrides don't need to do anything extra to their description, though they do.